fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/ServerEntityWorldChangeEvents.java

```diff
@@ -45,6 +45,19 @@ public final class ServerEntityWorldChangeEvents {
 		}
 	});
 
+	/**
+	 * Called before a player is being moved to a different world.
+	 */
+	public static final Event<AllowPlayerChange> ALLOW_PLAYER_CHANGE_WORLD = EventFactory.createArrayBacked(AllowPlayerChange.class, callbacks -> (player, origin, destination) -> {
+		for (AllowPlayerChange callback : callbacks) {
+			if (!callback.allowChangeWorld(player, origin, destination)){
```

Comment:
Checkstyle will fail:

```suggestion
			if (!callback.allowChangeWorld(player, origin, destination)) {
```

Reply:
thank you!